src/components/templates/Playa/Playa.tsx

```diff
@@ -132,9 +133,7 @@ const minZoom = () => (window.innerWidth - 2 * PLAYA_MARGIN_X) / PLAYA_WIDTH;
 const Playa = () => {
   useFirestoreConnect("venues");
   const [showModal, setShowModal] = useState(false);
-  const [selectedUserProfile, setSelectedUserProfile] = useState<
-    WithId<User>
-  >();
+  const [selectedUserProfile, setSelectedUserProfile] = useState<User>();
```

Comment:
Should I leave it as is until it is deleted?

Reply:
In general i would be interested to know your logic/reasoning in why it was changed. But I don't believe it will break anything currently so we don't need to revert it.

Reply:
Have explained it in a chat

Reply:
Yup, all made sense. It was basically an early failed attempt, then went a different direction in the other similar files. Since this file is getting deleted soon anyway, didn't make sense to put the effort into cleaning it up 'properly' here.